cpp/Platform.Timestamps/Timestamp.h

```diff
@@ -1,25 +1,73 @@
-﻿namespace Platform::Timestamps
-{
-    struct Timestamp : public IEquatable<Timestamp>
-    {
-        public: inline static std::string DefaultFormat = "yyyy.MM.dd hh:mm:ss.fffffff";
-
-        public: std::uint64_t Ticks = 0;
+﻿#pragma once
 
-        public: Timestamp(std::uint64_t ticks) { Ticks = ticks; }
+#include "CommonEraClock.h"
+#include <functional>
+#include <iomanip>
+#include <sstream>
+#include <string>
```

Comment:
in many codestyles the following order of includes is:
1) related header file
2) std
3) third party libraries
4) project header files

according this, includes should be like:
```cpp
#include <functional>
#include <iomanip>
#include <sstream>
#include <string>

#include "CommonEraClock.h"
```